src/consolelogger.c

```diff
@@ -203,16 +203,29 @@ void consolelogger_log(LOG_CATEGORY log_category, const char* file, const char*
     time_t t;
     va_list args;
     va_start(args, format);
+#if (defined(_MSC_VER) && defined(_DEBUG))
+    char* debugPrint;
+#endif
 
     t = time(NULL);
 
     switch (log_category)
     {
     case AZ_LOG_INFO:
         (void)printf("Info: ");
+#if (defined(_MSC_VER) && defined(_DEBUG))
+        OutputDebugStringA("Info: ");
+#endif
         break;
     case AZ_LOG_ERROR:
         (void)printf("Error: Time:%.24s File:%s Func:%s Line:%d ", ctime(&t), file, func, line);
+#if (defined(_MSC_VER) && defined(_DEBUG))
+        if ((debugPrint = printf_alloc("Error: Time:%.24s File:%s Func:%s Line:%d ", ctime(&t), file, func, line)) != NULL)
```

Comment:
>printf_alloc [](start = 26, length = 12)

sprintf_char does the same thing. Seems that this console logger should use that so that the code is not duplicated.

Reply:
Makes sense. In the end then we shouldn't have `vprintf_alloc` and `printf_alloc` in here since we already have a string_utils.